### DISCUSSION TITLE
what hyperparmeters are optimized using optuna?

### DISCUSSION THREAD

**Discussion by vnltbb** [optuna/optuna#6295] [Q&A]
hello everyone, 

i am a student researching methods for adopting CNN models for image-based diagnosis of plant diseases. 
for my research, i want to optimize hyperparameters for each model to compare their performance.

i am using Optuna to adjust setting model by model. i am particularly interested in understanding suitable hyperparameter settings such as epochs, batch size, learning rate and dropout rate. 

my questions are:
1. what hyperparameters are typically optimized when using transfer learning models?
2. can these hyperparameters be optimized using Optuna?
3. if the answer to 2. is no, how can i determine suitable settings for these hyperparameters? 

i understand that my questions my seem basic, as i am still building my foundational knowledge in this area. i would be grateful for any guidance, even if it covers fundamental concepts. if i have any misconceptions or incorrect understanding, please fell free to correct me.

**Comment by nzw0301** [comment]
1. The question sounds a bit vague, so I suggest checking research papers in CNN or transfer learning.
2. Of course, you can use optuna to optimise the hyper-parameter of neural networks.
